packages/sitecore-jss/src/utils/utils.ts

```diff
@@ -107,10 +107,18 @@ export const enforceCors = (
   res: OutgoingMessage,
   allowedOrigins?: string[]
 ): boolean => {
+  if (!req.headers.origin) {
+    return true;
```

Comment:
If origin is not present, we might need to return _false_. If you return _true_ it means that header is set and we allow request to proceed

Reply:
The origin header should only be present for CORS requests from a browser. The idea here is that we shouldn't try to enforce any CORS validation for non-CORS requests (e.g. those made server-side).

Reply:
@ambrauer Thanks Adam. Now it's clear. I would prefer to have some js comment here to make it clear, somebody else in a future might have the same question that I had